src/Homework0921Methods/SumOfDiagonalOfMatrix.java

```diff
@@ -0,0 +1,29 @@
+package Homework0921Methods;
+
+public class SumOfDiagonalOfMatrix {
+    /*Ունեք nxn չափի մատրիցա, n-ը որոշեք դուք։ Պետքա գրել ծրագիր, որը կհաշվի
+    մատրիցայի անկյունագծերի վրայի թվերի գումարը։*/
```

Comment:
կհաշվի
    մատրիցայի անկյունագծ**### _երի_** վրայի թվերի գումարը

Reply:
Ok,I got it,thanks.